cleanlab/multiannotator.py

```diff
@@ -275,26 +300,439 @@ def get_label_quality_multiannotator(
+    annotator_weight = multiannotator_info["annotator_weight"]
+    avg_annotator_weight = np.mean(annotator_weight)
+
+    # compute scores for labeled data
+    active_learning_scores = np.full(len(labels_multiannotator), np.nan)
+    for i in range(len(active_learning_scores)):
+        annotator_labels = labels_multiannotator.iloc[i]
+        active_learning_scores[i] = np.average(
+            (quality_of_consensus_labeled[i], 1 / num_classes),
+            weights=(
+                np.sum(annotator_weight[annotator_labels.notna()]) + model_weight,
+                avg_annotator_weight,
+            ),
+        )
+
+    # compute scores for unlabeled data
+    if pred_probs_unlabeled is None:
+        pred_probs_unlabeled = np.array([])
+
+    if len(pred_probs_unlabeled) > 0:
+        if pred_probs.shape[1] != pred_probs_unlabeled.shape[1]:
+            raise ValueError(
+                "pred_probs and pred_probs_unlabeled must have the same number of classes"
+            )
+
+        optimal_temp = find_best_temp_scaler(labels_multiannotator, pred_probs)
+        pred_probs_unlabeled = temp_scale_pred_probs(pred_probs_unlabeled, optimal_temp)
+        quality_of_consensus_unlabeled = np.max(pred_probs_unlabeled, axis=1)
+
+        active_learning_scores_unlabeled = np.average(
+            np.stack(
+                [
+                    quality_of_consensus_unlabeled,
+                    np.full(len(quality_of_consensus_unlabeled), 1 / num_classes),
+                ]
+            ),
+            weights=[model_weight, avg_annotator_weight],
+            axis=0,
+        )
+
     else:
-        return {
-            "label_quality": label_quality,
-        }
+        active_learning_scores_unlabeled = np.array([])
+
+    return active_learning_scores, active_learning_scores_unlabeled
+
+
+def get_active_learning_scores_ensemble(
+    labels_multiannotator: Union[pd.DataFrame, np.ndarray],
+    pred_probs: np.ndarray,
+    pred_probs_unlabeled: Optional[np.ndarray] = None,
+) -> Tuple[np.ndarray, np.ndarray]:
+    """Returns an active learning score for each example in the dataset. The score is in between 0 and 1; lower scores indicate less confidence on the examples
+    and that additional labels should be collected for those examples. For example:
+
+    * 1 - confident example, no additional labeling is required.
+    * 0 - unconfident example, it is suggested to collect more labels for this example.
+
+    This function allows for an ensemble of models to be trained and will use the aggregation of predictions from the multiple models to compute the active learning score.
+
+    Parameters
+    ----------
+    labels_multiannotator : pd.DataFrame of np.ndarray
+        2D pandas DataFrame or array of multiple given labels for each example with shape ``(N, M)``,
+        where N is the number of examples and M is the number of annotators.
+        ``labels_multiannotator[n][m]`` = label for n-th example given by m-th annotator.
```

Comment:
docstring should clarify this input has same format as expected by the `get_label_quality_multiannotator()` function.